diff --git a/dist/liquid.node.js b/dist/liquid.node.js
index 42b42e90385e939726d6959823aeaf2ce0671a6b..2f303c92f3f81cae6ec09ec8b18382c197c743b0 100644
--- a/dist/liquid.node.js
+++ b/dist/liquid.node.js
@@ -1610,7 +1610,7 @@ function newline_to_br(v) {
 function strip_html(v) {
     const str = stringify(v);
     this.context.memoryLimit.use(str.length);
-    return str.replace(/<script[\s\S]*?<\/script>|<style[\s\S]*?<\/style>|<.*?>|<!--[\s\S]*?-->/g, '');
+    return str.replace(/<script[\s\S]*?<\/script>|<style[\s\S]*?<\/style>|<.*?>|<!--[\s\S]*?-->/gs, '');
 }
 
 var htmlFilters = /*#__PURE__*/Object.freeze({
@@ -2202,7 +2202,7 @@ class Tokenizer {
     }
     *readFileNameTemplate(options) {
         const { outputDelimiterLeft } = options;
-        const htmlStopStrings = [',', ' ', outputDelimiterLeft];
+        const htmlStopStrings = [',', ' ', '\n', outputDelimiterLeft];
         const htmlStopStringSet = new Set(htmlStopStrings);
         // break on ',' and ' ', outputDelimiterLeft only stops HTML token
         while (this.p < this.N && !htmlStopStringSet.has(this.peek())) {
